backend-api/template.yaml

```diff
@@ -1321,11 +1333,12 @@ Resources:
         - !Ref PermissionsBoundary
         - !Ref AWS::NoValue
 
-  WellKnown5XXLowThresholdAlarm:    
+  WellKnown5XXLowThresholdAlarm:
+    Condition: DeployAlarms
     Type: AWS::CloudWatch::Alarm
     Properties:
       ActionsEnabled: true
-      AlarmDescription: "1% or more of requests are failing on the /.well-known/jwks.json endpoint"
+      AlarmDescription: "1% or more of requests are failing on the /.well-known/jwks.json endpoint. See runbook: https://govukverify.atlassian.net/wiki/spaces/DCMAW/pages/4800446694/Alarms+Runbook+Warnings"
```

Comment:
suggestion(non-blocking): parameterise the warning and alert runbook URLs

Reply:
Good idea - done